src/form-question-types.rst

```diff
@@ -924,54 +1026,52 @@ If adding images, note that the images are referenced in the choices sheet, and
  likert_widget,agree,Agree,agree.jpg
  likert_widget,strongly_agree,Strongly Agree,strongly_agree.jpg
 
+.. _select-from-map:
 
-.. _multi-select-widget:
+Select one from map widget
+"""""""""""""""""""""""""""
 
-Multi select widget
-~~~~~~~~~~~~~~~~~~~~~
+.. versionadded:: 2022.2.0
 
-type
-  :tc:`select_multiple {list_name}`
-appearance
-  *none*
+  `ODK Collect v2022.2.0 <https://github.com/getodk/collect/releases/tag/v2022.2.0>`_
 
-Multi select questions support multiple answers.
+.. warning::
+  The `map` appearance on selects currently only supports single selection of points and is not yet available in web forms (Enketo).
+
+  The different :ref:`basemap sources <mapping-settings>` currently have different performance. If Collect feels slow when creating the map or when selecting a choice, please describe what you are experiencing `on the forum <https://forum.getodk.org/c/support/6>`_. If you have many choices to include on a map, try a provider other than Google or Mapbox. You can also use a :ref:`choice filter <cascading-selects>` to reduce the number of choices that get mapped.
```

Comment:
More line breaks would be nicer. Before You can use a GeoJSON and When using a geometry column